### ISSUE TITLE
option for autostart

### ISSUE THREAD

**Issue by Martimlin** [tithanayut/UIDtoKeyboard#2]
nice work. 

Is it possible that all settings can be saved and the program can start with Windows and everything is "ready to scan"?

**Comment by fc117** [comment]
good idea, or a simple command line parameter to appli reading mode like `UIDtoKeyboard.exe /mode:2`

**Comment by marcelscholte** [comment]
need that too

**Comment by munch10** [comment]
I need this feature as well please

**Comment by mkapulica** [comment]
I have forked the project and changed and simplified some functionalities.
You can check it out and let me know if you need an option to save some settings and which ones you'd like to save:
[Link to project](https://github.com/mkapulica/NFC-UID-Keyboard-Emulator)

**Comment by munch10** [comment]
Thank you @mkapulica

**Comment by mkapulica** [comment]
I would just like to let you know that I've implemented the requested features in my fork of the project.
Hope you find it useful, even if it's been a year since this issue was opened.
